### PR TITLE
Remove VarDict

### PR DESCRIPTION
**Pull request by MaxUlysse**
Can't wait to do that for MuTect1 too.

### CODE REVIEW

**Comment by marcelm** [comment]
Can #344 be merged first? Otherwise I’ll have to fix conflicts again (and I was first! ;-) ).

**Comment by MaxUlysse** [comment]
No problem for me

File: main.nf [hunk 1]
```diff
@@ -612,10 +611,7 @@ if (verbose) bamsFMT2 = bamsFMT2.view {"Bams with Intervals for MuTect2: $it"}
 (bamsFFB, bamsAll, intervals) = generateIntervalsForVC(bamsAll, intervals)
 if (verbose) bamsFFB = bamsFFB.view {"Bams with Intervals for FreeBayes: $it"}
 
-// VarDict
-(bamsFVD, bamsAll, intervals) = generateIntervalsForVC(bamsAll, intervals)
-if (verbose) bamsFVD = bamsFVD.view {"Bams with Intervals for VarDict: $it"}
-
+// Manta and Strelka
```
Comment (marcelm) [inline main.nf]: That comment isn’t really necessary

Comment (MaxUlysse) [inline main.nf]: True, but I felt sad for Manta and Strelka, and I don't want them to be jealous of the other Variant Callers...
More seriously, I can totally remove that

**Pull request merged**